UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/SceneExporter/ExporterTextures.cs

```diff
@@ -225,6 +241,9 @@ private string GetImageOutputPath(Texture texture, TextureMapType textureMapType
 
 			switch (textureMapType)
 			{
+				case TextureMapType.CubeMap:
+					textureHasAlpha = true;
```

Comment:
Why is this set to true? Just to ensure PNG compression when the export heuristic is one? Don't think cubemaps regularly have alpha channels

Reply:
In my MX_reflection_probe implementation, I'm exporting the reflection probes as-is. Which uses an RGBM encoding which encodes the multiplier into the alpha. Probably need to handle different HDR encodings via some passed parameter.

https://github.com/matrix-org/thirdroom-unity-exporter/blob/main/Runtime/Scripts/MX_reflection_probes/MX_ReflectionProbesExtension.cs#L37

Reply:
I think in Needle Engine we shifted the cubemap blit out of UnityGltf for that reason (and export a regular linear texture) - it's weird to assume all cubemaps are RBGM, more likely they're not.
Have you tried that?